src/ParallelAlgorithms/Events/ObserveFields.hpp

```diff
@@ -140,10 +140,7 @@ class ObserveFields<VolumeDim, ObservationValueTag, tmpl::list<Tensors...>,
         "An optional mesh to which the variables are interpolated. This mesh "
         "specifies any number of collocation points, basis, and quadrature on "
         "which the observed quantities are evaluated. If no mesh is given, the "
-        "results will be evaluated on the mesh the simulation runs on. "
-        "The user may add several ObserveField Events e.g. with and without an "
-        "interpolating mesh to output the data both on the original mesh and "
-        "on a new mesh.";
+        "results will be evaluated on the mesh the simulation runs on.";
```

Comment:
I think this note is helpful in this context, and if I understood @nilsdeppe correctly he didn't object, just suggested we write more documentation about events in a dev guide. So please drop this fixup commit.